adafruit_ds18x20.py

```diff
@@ -66,7 +73,7 @@ class DS18X20:
 
     """
 
-    def __init__(self, bus, address):
+    def __init__(self, bus: "OneWireBus", address: int) -> None:
```

Comment:
We can add `adafruit-circuitpython-onewire` to the `requirements.txt` and use `from adafruit_onewire import OneWireBus` in the typing import block.  Then you won't need to use a string reference.

Reply:
1.  Thanks for the feedback.  I have been slogging through these, at a level of python knowledge that while, increasing, is still not much higher than 'beginner'.  Also trying to deduce types by reading code, which sometimes challenges my level of familiarity.  So I appreciate the time you took to provide usable feedback to the process other than 'jeez, give it a break', which I've wondered if either of you were approaching that point of frustration with me.

2. related to this issue:  the requirements.txt does include the onewire call, and the original import group includes "from adafruit_onewire.device import OneWireDevice".  So adding the suggested line to the try-else block imports makes pylint throw a complaint:
31:4: C0412: Imports from package adafruit_onewire are not grouped (ungrouped-imports) and I didn't think I should submit this and ignore it.  I should have brought this up in advance, probably, so I apologize for not seeking out advice beforehand.....

Reply:
No worries, the great thing about this project is that it's all about learning, whether it's using or developing CircuitPython!  I'm definitely not frustrated when people submit helpful improvements!  Definitely just thought you might find that tip helpful if you ever decide to help out with other projects as well!

As for the issue, that's normally pylint's way of saying "Hey, you probably should group these together for readability".  Which is normally absolutely the case, but since we're kind of making use of the way CircuitPython sheds annotations, we want to tell pylint that we want to explicitly do it this way.

You can disable it the same way you do `# pylint: disable=unused-import` but instead use `# pylint: disable=ungrouped-imports` on the line in the try/except block.